src/server/gameEngine/store/store.ts

```diff
@@ -0,0 +1,14 @@
+import { EngineChessGame } from './store.models'
+
+const gameStore = new Map<string, EngineChessGame>()
+
+export function getGameStore() {
+	return gameStore
+}
```

Comment:
Valid question. Just bad design, I think. At least there is room for improvement.

It would be nice if the store provided
`getGame` `addGame` `updateGame` and `deleteGame` and the games could not be mutated outside of the store. But that is something for the future !

Reply:
I don't know if it is a good practice, but I think you could also create a new game object, but keep reference to the history as the object is pretty small otherwise.

> It would be nice if the store provided `getGame` `addGame` `updateGame` and `deleteGame` and the games could not be mutated outside of the store. But that is something for the future !

This might be better than the object solution I gave below.